code/renderers/web-components/package.json

```diff
@@ -69,7 +69,7 @@
     "web-component-analyzer": "^1.1.6"
   },
   "peerDependencies": {
-    "lit-html": "^1.4.1 || ^2.0.0"
+    "lit-html": "^2.0.0"
```

Comment:
V2 is consumed as `lit` and it would be good if we made that change here, too!

Reply:
Done! Thanks! 🙏